.github/workflows/ci.yml

```diff
@@ -58,16 +60,23 @@ jobs:
       - run: gem install pkg/*.gem
 
       - run: rake test
+        # Don't run specs on TruffleRuby and JRuby
```

Comment:
"spec" isn't a suitable word for us. "test" is better because we're using test-unit.

Could you add a comment why we don't run tests on TruffleRuby and JRuby?

Reply:
Done